### ISSUE TITLE
bugs

### ISSUE THREAD

**Issue by g7755725** [MatthewBooth/OTAUpdates#16]
The OTA Updates appvstill force closes on android v5.1.1 CM12.1  when selecting addons, the app crashes, when you select changelog for rom the app crashes and even not doing anything the app crashes can you fix this?

**Comment by MatthewBooth** [comment]
Not without any kind of logs to show me what's going on, to be honest. 

I'm running this on CM 12.1 myself, everything working, so we need to find the cause. Get mr some logcats of the crashes, and the links of any XMLs (if not hosted on romhut).

**Comment by g7755725** [comment]
I sent you a logcat via email on the email you provided on google play store even with the screen shot too

**Comment by MatthewBooth** [comment]
Ok, I'll check that out and get back to you.

**Comment by MatthewBooth** [comment]
I replied. It's not a whole log. Logs should be hundreds, if not thousands of lines.

**Comment by MatthewBooth** [comment]
Ok, I found the problem

In your ROM you seem to have included the APK manually instead of building from source. That's ok, but you needed to follow the building instructions and open the APK with an archive manager (7zip, winrar, etc) and find libbypass.so and put it into /system/lib/ in your ROM.

If you are building from source, I highly recommend adding the project to your local_manifest.sml and building that way, as source build versions of this app work best with CM or AOSP.

In short: Open the file called CMUpdater.apk in /system/priv-app/CMUpdater (your name) and take the libbypass.so file from the lib/arm-v7a folder and put it in /system/lib of your ROM.
